sensor/common/processfilter/filter.go

```diff
@@ -12,14 +13,16 @@ const (
 var (
 	singletonInstance sync.Once
 
+	maxUniqueProcesses = env.ProcessFilterMaxProcessPaths.IntegerSetting()
```

Comment:
Isn't this backwards?  Shouldn't this one be max int and the one in `central/processindicator/filter/singleton.go` use the env var?